packages/wells/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "private": true,
+  "name": "@cognite/sdk-wells",
+  "license": "Apache-2.0",
+  "repository": "cognitedata/cognite-sdk-js",
+  "homepage": "https://github.com/cognitedata/cognite-sdk-js#readme",
+  "main": "dist/index.js",
+  "types": "dist/src/index.d.js",
+  "version": "0.0.0",
+  "scripts": {
+    "clean": "rm -rf dist/ docs/ codeSnippets/",
+    "test": "jest --config=../../jest.config.js --testPathPattern=/wells/",
+    "lint": "eslint 'src/**/*.{js,ts}'",
+    "lint:fix": "yarn lint --fix",
+    "prepublishOnly": "yarn build",
+    "build": "yarn clean && rollup -c && yarn esCheck",
+    "watch": "rollup -cw",
+    "esCheck": "es-check es5 './dist/index.js'",
+    "docs": "typedoc --options typedoc.js --tsconfig tsconfig.json src/index.ts",
+    "docs:bundle": "yarn docs && mkdir -p ../../docs/wells && cp -r docs/* ../../docs/wells/",
+    "extract-snippets": "rm -rf codeSnippets/ && yarn docs --json codeSnippets/docs.json && node ../../scripts/extractCodeSnippets.js",
+    "test-snippets": "yarn extract-snippets && yarn tsc -p codeSnippets/tsconfig.build.json"
+  },
+  "dependencies": {
+    "@cognite/sdk": "^3.0.0",
+    "@cognite/sdk-core": "^1.0.0"
+  },
+  "publishConfig": {
+    "access": "public"
```

Comment:
I didn't notice you are going to publish it already.
Are you ok with the name? Could it be something more descriptive?

Reply:
@rek @kiril-me @jucc any input here?

Reply:
also if you want to publish it `"private"` supposed to be set to `false`

Reply:
I think we can keep "private": true and "access": "private" for now. And publish once the SDK has some use to it.

Reply:
From what I read, it would be enough to set "private": true? 
![image](https://user-images.githubusercontent.com/25263842/92712936-a475a100-f35a-11ea-8847-e6f80deb6a4a.png)

Reply:
@Sollimann 

> From what I read, it would be enough to set "private": true?

you might be right, it's ok then.